vulcanai/models/metrics.py

```diff
@@ -805,13 +805,8 @@ def bootfold_p_estimate(network, data_loader, n_samples, k, epochs,
 
         tot_num_imprv = float(len(ls_imprv_scores))
         score_abv_zero = 0.0
-        score_blw_zero = 0.0
-        for val in ls_imprv_scores:
-            if val > 1.0:
-                score_abv_zero += 1.0
-            else:
-                score_blw_zero += 1.0
-        p_val = float(float(score_abv_zero)/float(score_blw_zero))
+        score_avc_zero = sum(val > 1.0 for val in ls_imprv_scores)
```

Comment:
why is this score_avc_zero never used? it's score_abv_zero below. Is this intentional or a typo?

Reply:
That seems like a type, but I'll look into it.

Reply:
Fixed it, it was a typo.